core/modules/czar/python/app.py

```diff
@@ -340,7 +339,8 @@ def __init__(self, query, context, setSessionId, resultName=""):
         pass
 
     def _reportError(self, chunkId, code, message):
-        logger.dbg("reporting", chunkId, code, message)
+        # TODO remove lsst/log wrapper
```

Comment:
we should do it in a new ticket, why was this wrapper implemented? FYI, it seems it only concatenate its parameters and send it to lsst/log. If lsst/log API is stable now, removing the wrapper would simplify the code.